app-resource-manager/deployments/app-resource-manager/Chart.yaml

```diff
@@ -16,11 +16,9 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
-version: 2.4.4
+version: 2.4.5
```

Comment:
I think this should be `-dev` too like the others. But why not make a release `2.4.5` out of it?